gremlin-test/src/main/resources/org/apache/tinkerpop/gremlin/structure/io/script/script-input.groovy

```diff
@@ -1,47 +0,0 @@
-import org.apache.tinkerpop.gremlin.structure.VertexProperty
```

Comment:
any reason this was removed? still relevant to hadoop testing, no?

Reply:
reverted